src/components/Checkbox/index.tsx

```diff
@@ -96,10 +77,12 @@ const RealCheckbox = styled.input`
   display: none;
 `;
 
-const FlexContainer = styled.div`
+const FlexContainer = styled.div.attrs(({ withInputInfo = false }: { withInputInfo: boolean }) => ({
```

Comment:
why are you using `attrs` here?

Reply:
why not?

Reply:
Please read the documentation 
https://www.styled-components.com/docs/faqs#when-to-use-attrs